source/GameLoadingPanel.cpp

```diff
@@ -36,14 +44,20 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 
 #include "opengl.h"
 
-
+namespace {
+	std::string hint;
+}
 
 GameLoadingPanel::GameLoadingPanel(PlayerInfo &player, const Conversation &conversation,
 	UI &gamePanels, bool &finishedLoading)
 	: player(player), conversation(conversation), gamePanels(gamePanels),
 		finishedLoading(finishedLoading), ANGLE_OFFSET(360. / MAX_TICKS)
 {
 	SetIsFullScreen(true);
+
+	// Choose the loading hint.
+	std::vector<std::string> loadingHintArray = LoadingHints().loadingHintArray;
+	hint = loadingHintArray[Random::Int(sizeof(loadingHintArray))];
```

Comment:
```suggestion
	hint = LoadingHints().loadingHintArray[Random::Int(sizeof(LoadingHints().loadingHintArray))];
```

no need for a local copy